modules/services/nix-gc.nix

```diff
@@ -110,9 +110,10 @@ in {
       systemd.user.services.nix-gc = {
         Unit = { Description = "Nix Garbage Collector"; };
         Service = {
-          ExecStart = "${nixPackage}/bin/nix-collect-garbage ${
+          ExecStart = ''
+            ${pkgs.bash}/bin/bash -c "${nixPackage}/bin/nix-collect-garbage ${
               lib.optionalString (cfg.options != null) cfg.options
-            }";
+            }"'';
```

Comment:
It would probably be better to create a proper script for this to avoid potential quote escaping issues. Something like

``` nix
ExecStart = toString (pkgs.writeShellScript "nix-gc" ''
  exec ${nixPackage}/bin/nix-collect-garbage ${
    lib.optionalString (cfg.options != null) cfg.options
  }";
'');
```

should pretty closely match what is inside the NixOS module.